lib/metadataTypes/Asset.js

```diff
@@ -101,7 +101,7 @@ class Asset extends MetadataType {
         }
         Util.logger.info(`Downloaded: ${this.definition.type} (${Object.keys(metadata).length})`);
 
-        return { metadata: metadata, type: this.definition.type };
+        return { metadata: metadata[Object.keys(metadata)[0]], type: this.definition.type };
```

Comment:
Also wondering if we shouldnt be doing this earlier? 
```suggestion
        return { metadata: Object.values(metadata)[0], type: this.definition.type };
```